src/demo/rewireEnterpriseDataApiService.js

```diff
@@ -6,7 +6,20 @@ const rewire = () => {
     const pageSize = options.page_size || 50;
     const page = options.page || 1;
     const start = (page - 1) * pageSize;
-    const results = enrollments.slice(start, start + pageSize);
+
+    // A comparator function that uses the options.ordering field to determine sort order
+    const sortByOptions = (a, b) => {
+      const ordering = options.ordering || 'user_email';
+      const isDesc = ordering.startsWith('-');
```

Comment:
Confirmed that we are using the `babel-polyfill` so `startsWith` should be safe to use: https://github.com/edx/edx-portal/blob/682a4d72d181ea129c0c693f8bcc0d4e67dddb31/src/index.jsx#L1